cairo-contracts/packages/clients/src/cometbft/component.cairo

```diff
@@ -470,6 +508,24 @@ pub mod CometClientComponent {
             self.client_sequence.write(client_sequence);
         }
 
+        fn write_update_height(
+            ref self: ComponentState<TContractState>, client_sequence: u64, update_height: Height
+        ) {
+            let mut update_heights = self.update_heights.read(client_sequence);
+
+            if update_heights.len() == 255 {
+                update_heights.pop_front().unwrap();
```

Comment:
I think it is fine to do this, as it is not good to store all past consensus state heights here. We can even limit it to something like 50~100, as the main purpose for this function is mainly to help resolve recent race conditions between multiple relayers.

Reply:
Alright. Set to 100.
[4c5a1a3](https://github.com/informalsystems/ibc-starknet/pull/106/commits/4c5a1a3324d38af51e7376403b013b66834acd98)